fvtest/porttest/si.cpp

```diff
@@ -1023,6 +1026,7 @@ TEST(PortSysinfoTest, sysinfo_test_sysinfo_set_limit_CORE_FILE)
 	}
 
 	reportTestExit(OMRPORTLIB, testName);
+	#endif
```

Comment:
This endif should have a comment indicating what condition it matches, and shouldn't be indented.
```suggestion
#endif /* defined(OSX) */
```